libkbfs/md_ops.go

```diff
@@ -383,14 +404,22 @@ func (md *MDOpsStandard) processRange(ctx context.Context, id TlfID,
 			}
 		}
 
-		irmd, err := md.processMetadataWithID(ctx, id, bid, handle, r)
+		irmd, c, err := md.processMetadataWithID(ctx, id, bid, handle, r)
```

Comment:
Yeah, I didn't do it this way because of the `CheckValidSuccessor` thing. Seems like a much bigger change.  But I'll play around with it and see how it goes.